Configurations/README

```diff
@@ -35,6 +39,54 @@ In each table entry, the following keys are significant:
                            upon which other targets can be built.
                            Explained further below.  [1]
 
+        policy          => The policy for the config target.  Can be
+                           one of: 
+
+                           "primary"    This is a config target
+                                        directly supported by the
+                                        OpenSSL project.
+                           "secondary"  This is a config target
+                                        supported by at least one
+                                        named OpenSSL team member.
+                           "community"  This is a config target
+                                        supported by at least one
+                                        named (self defined) member of
+                                        the OpenSSL community.
+                           "unknown"    This is a config target that
+                                        does not have a named
+                                        supporter.
+                           "deprecated" This is a config target that
+                                        has been deemed legacy and is
+                                        subject for future
```

Comment:
maybe "and could be removed in the future" is more clear?

Reply:
I actually meant to write "for future removal".  Would that be suitable enough?

Reply:
certainly!